{
	"about": "https://jvndb.jvn.jp/ja/contents/2016/JVNDB-2016-006713.html",
	"title": "Python HPACK ライブラリを使用してビルドされた HTTP/2 の実装におけるサービス運用妨害 (DoS) の脆弱性",
	"link": "https://jvndb.jvn.jp/ja/contents/2016/JVNDB-2016-006713.html",
	"description": "Python HPACK ライブラリを使用してビルドされた HTTP/2 の実装には、サービス運用妨害 (DoS) の脆弱性にされる脆弱性が存在します。\r\n\r\n本脆弱性は、“HPACK Bomb”攻撃と呼ばれています。",
	"identifier": "JVNDB-2016-006713",
	"references": [
		{
			"text": "http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2016-6581",
			"source": "CVE",
			"id": "CVE-2016-6581"
		},
		{
			"text": "http://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2016-6581",
			"source": "NVD",
			"id": "CVE-2016-6581"
		},
		{
			"text": "https://jvndb.jvn.jp/ja/cwe/CWE-399.html",
			"id": "CWE-399",
			"title": "リソース管理の問題(CWE-399)"
		}
	],
	"cpe": [
		{
			"text": "cpe:/a:python:python",
			"version": "2.2",
			"vendor": "Python Software Foundation",
			"product": "Python"
		}
	],
	"cvss": [
		{
			"version": "2.0",
			"score": "7.8",
			"type": "Base",
			"severity": "High",
			"vector": "AV:N/AC:L/Au:N/C:N/I:N/A:C"
		},
		{
			"version": "3.0",
			"score": "7.5",
			"type": "Base",
			"severity": "High",
			"vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
		}
	],
	"date": "2017-01-23T16:12+09:00",
	"issued": "2017-01-23T16:12+09:00",
	"modified": "2017-01-23T16:12+09:00"
}
